src/models/transactions/common.ts

```diff
@@ -111,8 +111,7 @@ export interface BaseTransaction {
   AccountTxnID?: string
   Flags?: number | GlobalFlags
   LastLedgerSequence?: number
-  // TODO: Make Memo match the format of Signer (By including the Memo: wrapper inside the Interface)
-  Memos?: Array<{ Memo: Memo }>
+  Memos?: Memo[]
```

Comment:
the `verifyMemo` needs to be fixed as well

Reply:
It should work as-is. I think we were originally just passing in `tx.Memos` into `isMemo`

Reply:
We had to modify the `Signer` one, so I suspect we'll have to modify the `Memo` one too. Can you add a test just to double check?